show/bgp_common.py

```diff
@@ -398,6 +403,13 @@ def show_routes(args, namespace, display, verbose, ipver):
                 error_msg = output
             print(error_msg)
             return
+
```

Comment:
If one of the bgp instances in the middle returns error. Output from the remaining instances wont be displayed ?

Reply:
Yes. That is the intention.  Most of the case when the FRR of that namespace reject the cmd is due to the parameter the user specified.  and if one name space rejects it, all other namespace will reject the same way so it is best to stop on first error.  Also, if in case the user thinks the issue is only specific to the failing name space but not on other name space, user can always use -n option to display from which namespace he likes to see.